app/lib/project/MetapolatorProject.js

```diff
@@ -635,6 +640,17 @@ define([
         return zipUtil.encode(false, mem_io, targetDirName, dataType);
     };
 
+    _p.getOTFInstance = function(masterName, project) {
+        var model = project.open(masterName)
+          , master = model.query('master#' + masterName)
+          , font
+          , exportController
+          ;
+        exportController = new OTFExportController(master, model, masterName);
+        font = exportController.do_export();
+        return new Buffer(Int8Array(font.toBuffer()));
```

Comment:
Is this working in NodeJS? All these binary containers are horrible for interoperability.

OK, it seems like opentype.js is not up to the task ... https://github.com/nodebox/opentype.js/blob/b5d04ae68d9c2aba92cc18c5cd93bf677bc43694/dist/opentype.js#L508